composer.json

```diff
@@ -96,6 +95,7 @@
         "post-autoload-dump": [
             "@install-tools"
         ],
+        "analyse-deps": "@php dev-tools/vendor/bin/composer-dependency-analyser",
```

Comment:
hmm, shouldn't this complain about `react/promise` ?
or what's the difference to old tools, which were also not complaining about it ?

Reply:
This package was removed from `composer.json` so there's nothing to complain about (anymore) 😊.

Reply:
> hmm, shouldn't this complain about `react/promise` ?

It does: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/18062545974/job/51400855215